lib/suivi-pcrs.js

```diff
@@ -1,9 +1,39 @@
 import {maxBy} from 'lodash-es'
 
-import HttpError from './http-error.js'
+async function request(url, options) {
+  const res = await fetch(`${url}`, options)
 
-const PUBLIC_URL = process.env.NEXT_PUBLIC_URL
-const PORT = process.env.NEXT_PUBLIC_PORT || 3000
+  if (res.status === 204) {
+    return res
+  }
+
+  if (!res.ok) {
+    const {message} = await res.json()
+    throw new Error(message)
+  }
+
+  return res.json()
+}
+
+function buildOptions(method, token, body) {
```

Comment:
J'aurais plutôt opté pour un objet en paramètre afin d'éviter de se retrouver avec des appels un peu cryptiques comme `buildOptions('GET', null, {foo: 'bar'})` par exemple.

```suggestion
function buildOptions({method, token, body}) {
```